pkg/apis/core/v1beta1/types.go

```diff
@@ -463,6 +463,8 @@ type CDIConfigSpec struct {
 	FilesystemOverhead *FilesystemOverhead `json:"filesystemOverhead,omitempty"`
 	// Preallocation controls whether storage for DataVolumes should be allocated in advance.
 	Preallocation *bool `json:"preallocation,omitempty"`
+	// InsecureRegistries is a list of TLS disabled registries
+	InsecureRegistries []string `json:"insecureRegistries,omitempty"`
```

Comment:
I was under the impression that CDIConfig is deprecated. Shouldn't this new field be in the `CDISpec` type instead?

Reply:
CDIConfig is not deprecated but now owned by CDISpec